src/styles/Css/App.css

```diff
@@ -1,74 +1,15 @@
-@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
-
 * {
   margin: 0;
   padding: 0;
   text-decoration: none;
 }
 
 body {
-  background: #308393;
-  font-size: 10px;
-  padding: 0.2rem;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: stretch;
-  white-space: nowrap;
-}
-
-body h2,
-body span,
-body h3 {
-  color: #03045e;
-  font-weight: 600;
-  font-size: 1em;
-}
-
-#root {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
-
-.card {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
-
-::placeholder {
-  color: #308393;
-  font-weight: 600;
-}
-
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
-}
-
-.logo:hover {
-  filter: drop-shadow(0 0 2em rgba(100, 108, 255, 0.6666666667));
-}
-
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em rgba(97, 218, 251, 0.6666666667));
-}
-
-.read-the-docs {
-  color: #caf0f8 !important;
-}
-
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-
-  to {
-    transform: rotate(360deg);
-  }
+  background-image: url("../../assets/quakeBg.jpg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  margin: 0;
+  padding: 0;
```

Comment:
* As stated in the [requirements ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)
* You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size and weight.
Layout: composition and space between elements.
* As currently your fonts are different from the ones in the design 
![image](https://github.com/GeekyHacks/EarthquakesTracker/assets/52497246/722924e3-be8e-4d6f-b92c-02653b0d2a88)